### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by erezrokah**

### CODE REVIEW

**Comment by erezrokah-test** [comment]
Cool

**Comment by erezrokah** [comment]
wow

**Comment by erezrokah-test** [comment]
LGTM

File: services/api-service/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@types/node": "^10.10.3",
     "@types/puppeteer": "^1.12.1",
     "aws-testing-library": "^1.0.0",
-    "axios": "^0.18.0",
+    "axios": "^0.18.1",
```
Comment (erezrokah) [inline services/api-service/package.json]: cool

Comment (erezrokah-test) [reply services/api-service/package.json]: yes

**Pull request merged**